go.py

```diff
@@ -0,0 +1,18 @@
+'''
+All Python classes will reside in this file
```

Comment:
Hmm. Not sure about if we want all classes to reside in one file. We might have a monolith.

Reply:
We could move files around. I put this in as the initial storage and open to suggestions for file structure.